example_test.go

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package credentials_test
+
+import (
+	"context"
+	"fmt"
+
+	credentials "github.com/oras-project/oras-credentials-go"
+	"oras.land/oras-go/v2/registry/remote/auth"
+)
+
+func ExampleNewStoreFromDocker() {
+	// NewStoreFromDocker returns a Store based on the default docker config file.
+	// See the documentation for details.
+	ds, err := credentials.NewStoreFromDocker(credentials.StoreOptions{AllowPlaintextPut: true})
+	if err != nil {
+		panic(err)
+	}
+
+	// save credentials into the store
+	err = ds.Put(context.Background(), "localhost:8080",
+		auth.Credential{
+			Username: "username-example",
+			Password: "password-example"})
+	if err != nil {
+		panic(err)
+	}
+
+	// get credentials from the store
+	cred, err := ds.Get(context.Background(), "localhost:8080")
+	if err != nil {
+		panic(err)
+	}
+	fmt.Println(cred)
+
+	// delete the credentials from the store
+	err = ds.Delete(context.Background(), "localhost:8080")
+	if err != nil {
+		panic(err)
+	}
+}
+
+func ExampleNewStoreWithFallbacks() {
+	// NewStoreWithFallbacks returns a new store based on the given stores.
+	// The first store is used as the primary store, and the rest as fallbacks.
+	// See the documentation for details.
+	primaryStore, err := credentials.NewStore("example/path/config.json", credentials.StoreOptions{AllowPlaintextPut: true})
+	if err != nil {
+		panic(err)
+	}
+	fallbackStore, err := credentials.NewStoreFromDocker(credentials.StoreOptions{AllowPlaintextPut: true})
```

Comment:
I think we can leave the StoreOptions empty. Since the credentials will never be put in the fallback store.